src/modules/hashivault/client.rs

```diff
@@ -1,13 +1,20 @@
 use crate::core::NovopsContext;
 
-use anyhow::Context;
-use vaultrs::client::{VaultClient, VaultClientSettingsBuilder };
+use anyhow::{Context, Error};
+use vaultrs::client::{Client, VaultClient, VaultClientSettingsBuilder};
 use url::Url;
 use async_trait::async_trait;
 use std::{ collections::HashMap, env, fs, path::{Path, PathBuf} };
-use vaultrs::{kv2, kv1, aws, api::aws::requests::GenerateCredentialsRequest};
+use vaultrs::{kv2, kv1, aws, auth, api::aws::requests::GenerateCredentialsRequest};
 use log::debug;
 use home;
+use crate::modules::hashivault::config::{HashiVaultAuth};
+
+
+const KUBERNETES_SA_JWT_PATH: &str = "/var/run/secrets/kubernetes.io/serviceaccount/token";
```

Comment:
It seems this file is always used by default, ignore the environment variable of the same name. Maybe you want to use env var here as well as default fle path?

Reply:
Yes, I fixed the code to use this constant if the environment variable doesn't exists.